lib/src/project_finder/project_finder.dart

```diff
@@ -12,7 +12,10 @@ class ProjectFinder {
   Future<List<Project>> find({
     @required String path,
     String name,
+    bool isRoot,
```

Comment:
I think about dedicated method `findRoot(@required String path)` because searching root can be more quickly and filtering during searching root probably isn't necessary.

Reply:
This way we can look for subprojects together with other filters. If I move that to a separate method we lose this functionality. If I add another method we duplicate the functionality :/.